recipe/meta.yaml

```diff
@@ -23,6 +23,8 @@ requirements:
     - setuptools >=45
     - setuptools-scm >=6.2
   run:
+    - fsspec
+    - venvception >=0.0.5
```

Comment:
This `venvception` library (added in https://github.com/pangeo-forge/pangeo-forge-runner/pull/167) will need to be packaged in conda-forge first at https://github.com/conda-forge/staged-recipes.